src/circuit.mjs

```diff
@@ -310,6 +310,19 @@ export class HeadlessCircuit {
         ret.subcircuits = subcircuits;
         return ret;
     }
+    getSvgExportUrl(circuitSvgDOMElement) {
+        // Remove useless in export jointJS tooltips
+        circuitSvgDOMElement
+            .querySelectorAll('foreignObject.tooltip')
+            .forEach(obj => obj.remove());
+
+        // Serialize SVG to string & encode it as URL
+        let data = (new XMLSerializer()).serializeToString(circuitSvgDOMElement);
+        data = '<?xml version="1.0" standalone="no"?>\r\n' + data;
+        const url = 'data:image/svg+xml;charset=utf-8,' + encodeURIComponent(data);
```

Comment:
Isn't this method doing too much? DigitalJS API user might want to get a non-URL-encoded SVG.

Reply:
That is a good suggestion. I will change it to return just the serialized SVG string.

Reply:
In the last commit I rewrote this method to return just the serialized SVG string and not the encoded URL.